### DISCUSSION TITLE
[Time series Echarts] Add option to show individual series values even when stacked

### DISCUSSION THREAD

**Discussion by kamalkeshavani-aiinside** [apache/superset#17222] [Q&A / Help] [answered]
**Is your feature request related to a problem? Please describe.**
Currently, when we stack series and select 'Show Value' option, the chart will show only total of series' at a point.

![image](https://user-images.githubusercontent.com/74634977/138622642-6b6ca3e7-4974-42b8-8154-883d0aa927ee.png)

**Describe the solution you'd like**
There should be an option to select whether to show total or individual series values in the chart.

Without stack:
![image](https://user-images.githubusercontent.com/74634977/138622751-225b8460-3881-4d3f-9d46-3825cf37eead.png)

**Describe alternatives you've considered**

**Additional context**
Here we will have issue of too many series', cluttering the value labels. For this I guess the best option is to show only values when there is enough space available.

Area chart sample:
![image](https://user-images.githubusercontent.com/74634977/138623772-52692203-110a-4860-9349-3e608d1e4bc4.png)

**Comment by stephenLYZ** [comment, marked as answer]
There is an `only_total` control that might suit your needs.

**Reply by kamalkeshavani-aiinside** [reply]
@stephenLYZ Where is this `only_total` control that you are talking about?

**Reply by stephenLYZ** [reply]
![image](https://user-images.githubusercontent.com/11830681/139114028-3d447136-bb8a-43ab-ac9e-2de03bbd5374.png)

Btw, Is it the latest version?

**Reply by kamalkeshavani-aiinside** [reply]
I don't see this option in latest released version 1.3.2. Is it there in master?

**Reply by kamalkeshavani-aiinside** [reply]
I confirmed it is present in master. Thank you!
